### PR TITLE
fix(VertexIdentifier): better MeshComponentVertex naming

### LINKED ISSUE #484: rename MeshComponentVertex -> ComponentMeshVertex

### PR DESCRIPTION
**Pull request by BotellaA**
BREAKING CHANGE: MeshComponentVertex and associated methods have been renamed in ComponentMeshVertex

fix #484

### CODE REVIEW

File: tests/model/test-vertex-identifier.cpp [hunk 1]
```diff
@@ -126,30 +126,30 @@ void test_set_unique_vertices(
...
         "[Test] Creation of unique vertices is not correct" );
 
-    const auto& uvertices0 = vertex_identifier.mesh_component_vertices( 0 );
+    const auto& uvertices0 = vertex_identifier.component_mesh_vertices( 0 );
     OPENGEODE_EXCEPTION( uvertices0.size() == 2,
         "[Test] Search of unique vertices is not correct" );
-    OPENGEODE_EXCEPTION( vertex_identifier.has_mesh_component_vertices(
+    OPENGEODE_EXCEPTION( vertex_identifier.has_component_mesh_vertices(
                              0, geode::Corner2D::component_type_static() ),
         "[Test] Unique vertex should have mesh component vertices of type "
```
Comment (panquez) [inline tests/model/test-vertex-identifier.cpp:134]:
```suggestion
        "[Test] Unique vertex should have component mesh vertices of type "
```
Maybe elsewhere

File: tests/model/test-vertex-identifier.cpp [hunk 2]
```diff
@@ -126,30 +126,30 @@ void test_set_unique_vertices(
...
         "Corner" );
-    const auto& uvertices0_type = vertex_identifier.mesh_component_vertices(
+    const auto& uvertices0_type = vertex_identifier.component_mesh_vertices(
         0, geode::Corner2D::component_type_static() );
     OPENGEODE_EXCEPTION( uvertices0_type.size() == 2,
         "[Test] Search of unique vertices by "
         "MeshComponentType is not correct" );
```
Comment (panquez) [inline tests/model/test-vertex-identifier.cpp:140]:
```suggestion
        "ComponentMeshType is not correct" );
```

Comment (panquez) [reply tests/model/test-vertex-identifier.cpp:140]: ?

**Review by panquez — APPROVED** [review]

File: tests/model/test-vertex-identifier.cpp [hunk 3]
```diff
@@ -126,30 +126,30 @@ void test_set_unique_vertices(
...
+        vertex_identifier.has_component_mesh_vertices( 0, uuids[0] ),
+        "[Test] Unique vertex should have component mesh vertices of given "
         "Corner id" );
     OPENGEODE_EXCEPTION(
-        vertex_identifier.mesh_component_vertices( 0, uuids[0] ).size() == 1
-            && vertex_identifier.mesh_component_vertices( 0, uuids[0] ).front()
+        vertex_identifier.component_mesh_vertices( 0, uuids[0] ).size() == 1
+            && vertex_identifier.component_mesh_vertices( 0, uuids[0] ).front()
                    == 0,
         "[Test] Search of unique vertices by MeshComponent UUID is not "
```
Comment (MelchiorSchuh) [inline tests/model/test-vertex-identifier.cpp:149]: ComponentMesh

**Review by MelchiorSchuh — APPROVED** [review]

**Pull request merged**